client/src/theme/overrides/Button.js

```diff
@@ -105,6 +105,28 @@ export default function Button(theme) {
             opacity: 0.5,
           },
         },
+        gray: {
+          padding: "6px 12px",
+          color: '#313233',
+          backgroundColor: theme.palette.common.white,
+          borderStyle: "solid",
+          borderWidth: "1px",
+          borderColor: '#999999',
+          fontSize: ".8125rem",
+          "@media (min-width:600px)": {
+            fontSize: "0.8125rem",
+          },
+          "&:hover": {
+            backgroundColor: '#E1E1E1',
```

Comment:
The background color should be "#E2E5EE"  instead of "#E1E1E1" according the Figma.

<img width="848" alt="Screenshot 2024-01-26 at 4 35 33 PM" src="https://github.com/hackforla/food-oasis/assets/68657634/a2daeb32-7599-4f74-88b1-087af1294546">

Reply:
Omg thats so helpful, I didn't see that in the sigma and I made up the hover state! I wasn't to far from the real one haha